README.md

```
@@ -90,60 +141,77 @@ pre-commit install
+   1. 啟動服務
+      ```bash
+      make rebuild
+      ```
+
+   2. 建立管理員帳號
+      ```bash
+      make manage cmd="createsuperuser"
+      ```
+
+可存取的微服務：
+   - Django Admin: <http://localhost:8000/admin>
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 0; Actual: 3

(MD007, ul-indent)